lib/aws/active_job/sqs/lambda_handler.rb

```diff
@@ -5,62 +5,65 @@
 module Aws
   module ActiveJob
     module SQS
-      class << self
-        # A lambda event handler to run jobs from an SQS queue trigger.
-        # Configure the entrypoint to: +config/environment.Aws::ActiveJob::SQS.lambda_job_handler+
-        # This will load your Rails environment, and then use this method as the handler.
-        def lambda_job_handler(event:, context:)
-          return 'no records to process' unless event['Records']
+      # Lambda event handler to run jobs from an SQS queue trigger
+      module LambdaHandler
```

Comment:
Maybe remove handler from name or method? Up to you.

Reply:
I went back and forth on this name a lot.  An `SQS::Lambda` module felt weird so I added the "Handler" to it.  And the method name reads a bit strangely without the word "handler" or "handle" in it.  My other idea was `handle_job_event`, but I think `job_handler` fits better with lambda's naming of things.